lib/shared/addon/mixins/reservation-check.js

```diff
@@ -197,4 +197,26 @@ export default Mixin.create({
 
     return errors
   },
+
+  updatePreRequest() {
+    const answers = get(this, 'app.answers') || {};
+    const workloads = this.getEnalbedWorkloads(answers);
+
+    const preRequestsCpu = workloads.reduce((all, current) => {
+      const value = answers[`${ current }.resources.requests.cpu`]
+
+      return value ? all + convertToMillis(value) : all
+    }, 0)
+
+    const preRequestsMemory = workloads.reduce((all, current) => {
+      const value = answers[`${ current }.resources.requests.memory`]
+
+      return value ? all + parseSi(value) / 1048576 : all
+    }, 0)
+
+    setProperties(this, {
```

Comment:
Have you considered refactoring this function to take input and return output rather than modify data directly on the consumer of the mixin? Or perhaps giving this method a more verbose name like `updateCpuMemoryPreRequest`? Mixin's can be somewhat nebulous and it is not very clear what this function does from both places it's called.